libs/payments/paypal/src/lib/paypal.manager.in.spec.ts

```diff
@@ -225,6 +225,17 @@ describe('PaypalManager', () => {
       );
       expect(result).toEqual(expected);
     });
+
+    it('returns empty array when no subscriptions', async () => {
+      const mockCustomer = StripeCustomerFactory();
+
+      stripeManager.getSubscriptions = jest.fn().mockResolvedValueOnce([]);
```

Comment:
I don't think the mock value lines up correctly with the return type for getSubscriptions